README.md

```diff
@@ -1,5 +1,5 @@
 # Solar2D Game Engine
-## rebranded Corona CoronaSDK or Corona
+## Rebranded Corona SDK or Corona
```

Comment:
Maybe Corona SDK with a space would be better

Reply:
Thanks. I agree